Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -10826,7 +10829,9 @@ def xsiam_api_call_with_retries(
     zipped_data,
     headers,
     num_of_attempts,
-    events_error_handler=None
+    error_msg='',
+    events_error_handler=None,
```

Comment:
Let's keep the original order of the args.
```suggestion
    events_error_handler=None,
    error_msg='',
```